orx-palette/src/main/kotlin/PaletteStudio.kt

```diff
@@ -201,9 +202,8 @@ class PaletteStudio(
     }
 
     fun randomPalette() {
-        val comparison = palette.colors.toMutableList()
+        val comparison = mutableListOf(palette.colors)
```

Comment:
This was actually suggested by @ricardomatias 
Without this the algorithm was sometimes returning the same palette twice in a row